README.md

```diff
@@ -43,7 +43,7 @@ If you wish to switch to Trello, do the following:
 5. Set the following values in the .env file:
         TRELLO_KEY=trello-key
         TRELLO_TOKEN=trello_token
-        TRELLO_WORKSPACE_NAME=todoapp
+        TRELLO_BOARD_NAME=todoapp
```

Comment:
I have to be honest, I thought this would happen automatically.
Where can I see what dependencies I have installed?

Reply:
@BenRamchandani 
I'm struggling to get "test_task_journey" to pass when they run together. Running individually it works fine, so I suspect some global state is being shared.
To be honest I'm struggling to debug because the pytest output is verbose when it fails and nothing when it succeeds.
I want to add some logging to a log file, any suggestions of how to do this?

Reply:
Hey, sorry for missing this.

* There's a list of dependencies in `pyproject.toml`
* If you run `poetry run pytest --log-cli-level=INFO` you'll get log output from pytest which can be helpful

Reply:
thanks @BenRamchandani - this is what I was struggling with, I couldn't see the log output in the console during the tests, much appreciated!